cookbook/deployment/cluster/auth_setup.py

```diff
@@ -1,8 +1,518 @@
 """
+Authentication in Flyte
```

Comment:
This file should be auth_setup.rst

Reply:
`auth_setup.py` is correct, see https://github.com/flyteorg/flytesnacks/pull/245 which explains why we need to maintain a consistent structure of `README.rst` and `*.py` files in sphinx gallery example directories.